spec/models/file_search_spec.rb

```diff
@@ -13,16 +13,35 @@
   let(:catalog_query) { double("Catalog query", search_params_logic: logic) }
   let(:file_query) { double("File query") }
   let(:files) { [double("GenericFile1"), double("GenericFile2")] }
+  let(:response) { double("Response",
+                    total_pages: 2,
+                    current_page: 1,
+                    next_page: 2,
+                    response: { "numFound" => 20 }
+                    ) }
 
   context "with no params" do
-    let(:params) { {} }
+    let(:params) { { } }
+    let(:expected_query) { { "f": {"highlighted_sim"=>"1"} } }
```

Comment:
Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.